### DISCUSSION TITLE
Track time job stays in queue

### DISCUSSION THREAD

**Discussion by marklynk** [taskforcesh/bullmq#706] [Q&A] [answered]
Hi, is there any built in mechanism for tracking how long a job was queued before being picked up by a worker?

**Comment by manast** [comment, marked as answer]
There are two job properties you can use for tracking this: ```timestamp``` and ```processedOn```: https://github.com/taskforcesh/bullmq/blob/master/docs/gitbook/api/bullmq.job.md

**Reply by marklynk** [reply]
Perfect, thank you!
